crates/top/re_sdk/src/web_viewer.rs

```diff
@@ -120,6 +120,7 @@ pub fn host_web_viewer(
     bind_ip: &str,
     web_port: WebViewerServerPort,
     force_wgpu_backend: Option<String>,
+    video_decoder: Option<String>,
     open_browser: bool,
     source_url: &str,
```

Comment:
This parameter list is getting out of hand; we should use the builder-pattern instead

Reply:
I'm not convinced it is. But at this time of day easier to implement it than to argue my point. I put it into a different PR since it's a user facing breaking change that I don't want to have in here.

https://github.com/rerun-io/rerun/pull/7553